spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -346,32 +344,36 @@ Sort criteria are provided statically for a repository method by using the `Orde
 
 ==== Dynamic Mechanisms for Sort Criteria
 
-Sort criteria are provided dynamically to repository methods either via `Sort` parameters or via a `PageRequest` or `Order` parameter that has one or more `Sort` values. `Sort` and `PageRequest` containing `Sort` must not both be provided to the same method. Similarly, `Order` and `PageRequest` containing `Sort` must not both be provided to the same method.
+Sort criteria are provided dynamically to repository methods either via `Sort` parameters or via a `Order` parameter that has one or more `Sort` values.
 
 ==== Examples of Sort Criteria Precedence
 
 The following examples work through scenarios where static and dynamic sort criteria are provided to the same method.
 
 [source,java]
 ----
-// Sorts first by type. When type is the same, applies the PageRequest's sort criteria
-Page<User> findByNameStartsWithOrderByType(String namePrefix, PageRequest<User> pagination);
+// Sorts first by type. When type is the same, applies the Order's sort criteria
+Page<User> findByNameStartsWithOrderByType(String namePrefix,
+                                           PageRequest pagination,
+                                           Order<User> sorts);
```

Comment:
We don't have to do it in this PR, but it would be better IMO, to rewrite the examples in this section to use something other than Query by Method Name.

Reply:
Yes, that would be nice, but it is definitely out of scope for this PR. It could even be done as a compatible change post v1.0 when we add some good alternatives.